src/lib/Containers/Conversation.tsx

```diff
@@ -86,21 +91,12 @@ export class Conversation extends React.Component<RelayProps, any> {
   }
 
   render() {
-    const conversation = this.props.me.conversation
+    const conversation = this.props.viewer.me.conversation
     const partnerName = conversation.to.name
-    const artwork = conversation.artworks[0]
-    const temporaryTimestamp = "11:00AM"
-
-    // Ideally we will use a Relay fragment in the Message component, but for now this is good enough
-    const messageData = conversation.messages.edges.reverse().map(({ node }, index) => {
-      return {
-        senderName: this.isFromUser(node) ? conversation.from.name : partnerName,
-        key: index,
```

Comment:
In `ListView`s and `FlatList`s, every item has to have a unique `key`. Usually we just use the `id` or `__id` of the object; whichever one is numeric.